src/main/java/nextstep/courses/domain/session/Session.java

```diff
@@ -0,0 +1,62 @@
+package nextstep.courses.domain.session;
+
+import nextstep.courses.CannotEnrollStateException;
+import nextstep.courses.domain.course.Course;
+import nextstep.users.domain.NsUser;
+
+import java.time.LocalDate;
+
+public class Session {
+
+    private Long id;
+
+    private CoverImg coverImg;
+
+    private SessionStatus sessionStatus;
+
+    private Period period;
+
+    private SessionUsers sessionUsers;
+
+    private Course course;
+
+    public Session() {}
+
+    public Session(CoverImg coverImg, boolean isFree, SessionStatus sessionStatus, Integer maxAttendance, Course course, Period period) {
+        this.coverImg = coverImg;
+        this.sessionStatus = sessionStatus;
+        this.sessionUsers = new SessionUsers(isFree, maxAttendance);
+        this.course = course;
+        this.period = period;
+    }
+
+    public Session(CoverImg coverImg, boolean isFree, SessionStatus sessionStatus, Course course, Period period) {
+        this(coverImg, isFree, sessionStatus, null, course, period);
+    }
+
+    public static Session notFreeSession(CoverImg coverImg, int maxAttendance, Course course, Period period) {
+        if (!period.isValidDate(course)) {
+            throw new IllegalArgumentException("유효하지 않는 세션 일정입니다.");
+        }
+        return new Session(coverImg,false, SessionStatus.PREPARE, maxAttendance, course, period);
+    }
+
+    public static Session FreeSession(CoverImg coverImg, Course course, Period period) {
```

Comment:
```suggestion
    public static Session freeSession(CoverImg coverImg, Course course, Period period) {
```

정적팩토리 메소드라서 소문자로 시작하는게 맞는것 같아요 🤔

Reply:
감사합니다!! notFreeSession이랑 같이 작업을 하다 실수로 대문자로 쓴것 같네요 ㅠㅠ